dbcon/execplan/calpontsystemcatalog.cpp

```diff
@@ -2992,6 +2990,12 @@ const CalpontSystemCatalog::RIDList CalpontSystemCatalog::columnRIDs(const Table
         checkSysCatVer();
     }
 
+    if (lower_case_table_names)
+    {
+        boost::algorithm::to_lower(aTableName.schema);
+        boost::algorithm::to_lower(aTableName.table);
+    }
```

Comment:
It's fine where it is because all that happens in between are checks not relying on the case of aTableName. However, in the tiny chance that someday we change the case of calpontsystemcatalog, I will move it.